### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_Keyboard: Keyboard focus not clearly visible for several interactive elements available on the screen in Chrome.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2827]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 2.4.7

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate to all the interactive elements of "Download Insiders" screen.
5.	Observe if the keyboard focus is clearly visible for not.

**Actual:**
In Chrome browser when users navigate to the several interactive elements using keyboard the keyboard focus should be clearly visible to the keyboard users.

**Expected:**
The keyboard focus should be clearly visible when keyboard users navigates through the several interactive elements.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the keyboard focus is not clearly visible when users navigates through the interactive elements then the users will get confused and might not be able to interact with the elements.

**MAS Reference:**
MAS 2.4.7 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

**Attachment for Reference:**
[Code.visualstudio.com_DownloadInsiders_Keyboard Keyboard focus not clearly in Chrome..pptx](https://github.com/microsoft/vscode-docs/files/3338169/Code.visualstudio.com_DownloadInsiders_Keyboard.Keyboard.focus.not.clearly.in.Chrome.pptx)

**Comment by octref** [comment]
Good catch. Previously:

![image](https://user-images.githubusercontent.com/4033249/60848868-32755b00-a19d-11e9-878e-50466909173e.png)

With fix:

![image](https://user-images.githubusercontent.com/4033249/60848877-3acd9600-a19d-11e9-8d15-b3621fb11561.png)
